### ISSUE TITLE
Multi parent

### ISSUE THREAD

**Issue by sergiogomesamkor** [bumbeishvili/d3-organization-chart#59]
Hello, 
Can you give me a tip on how to customize this solution for a node to support multi parent.

**Comment by bumbeishvili** [comment]
FYO, in the new, coming version, we will have multi links functionality. You will be able to specify connections with labels between any nodes

https://github.com/bumbeishvili/d3-organization-chart/blob/ee33e5ed64afa0be1ca2c695ed4d525aae67f98e/tree.html#L138

It will look like this

<img width="1672" alt="Screenshot 2021-07-29 at 17 46 20" src="https://user-images.githubusercontent.com/6873202/127504411-b50ab9a4-493a-4e4f-84b1-ce4df2f6e8c2.png">

**Comment by bumbeishvili** [comment]
It got merged - https://github.com/bumbeishvili/org-chart/pull/70

**Comment by TildeWill** [comment]
Is there a way to modify the HTML/SVG for the connection? For example, I want to make the line dotted and not have an arrow head.

**Comment by bumbeishvili** [comment]
Yes, you can override `connectionsUpdate` method and just uncomment relevant lines

https://stackblitz.com/edit/web-platform-nrybqp

```javascript
chart.connectionsUpdate(function (d, i, arr) {
            d3.select(this)
              .attr('stroke', (d) => '#152785')
              .attr('stroke-linecap', 'round')
              .attr('stroke-width', (d) => '5')
              .attr('pointer-events', 'none')

              //  .attr("marker-start", d => `url(#${d.from + "_" + d.to})`)
              //  .attr("marker-end", d => `url(#arrow-${d.from + "_" + d.to})`)
              .attr('stroke-dasharray', '20, 20');
          })
```

<img width="864" alt="Screenshot 2021-09-26 at 14 27 41" src="https://user-images.githubusercontent.com/6873202/134804029-108b52ce-ef5f-41ce-ab7e-1f5468fb151d.png">

**Comment by TildeWill** [comment]
Perfect! Any way to change the anchor points to the sides of the boxes rather than the top and bottom?

**Comment by bumbeishvili** [comment]
I don't think so

This is the relevant line if you decide to change source code by yourself

https://github.com/bumbeishvili/org-chart/blob/874bae3d9f5cc1f0da19d69ae67ad319966d82ba/src/d3-org-chart.js#L775
